rdkit/Chem/Draw/SimilarityMaps.py

```diff
@@ -121,12 +116,12 @@ def GetStandardizedWeights(weights):
     Parameters:
       weights -- the list with the atomic weights
     """
-  tmp = [math.fabs(w) for w in weights]
-  currentMax = max(tmp)
-  if currentMax > 0:
-    return [w / currentMax for w in weights], currentMax
-  else:
-    return weights, currentMax
+
+  maxAbsWeight = max([math.fabs(w) for w in weights])
```

Comment:
You don't need the square brackets here. 
```
maxAbsWeight = max(math.fabs(w) for w in weights)
```

Reply:
If that so, there is a tons of places use this style. But I simply do not have time to look over them

Reply:
That's ok. Just don't do that in your changes.

Reply:
OK, I got your idea! I would fix it in the next patch. Thanks you for your review